go/protocol/keybase1/extras.go

```diff
@@ -1905,6 +1905,20 @@ func (s SigChainLocation) LessThanOrEqualTo(s2 SigChainLocation) bool {
 	return s.SeqType == s2.SeqType && s.Seqno <= s2.Seqno
 }
 
+func (s SigChainLocation) Comparable(s2 SigChainLocation) error {
+	if s.SeqType != s2.SeqType {
+		return fmt.Errorf("mismatched seqtypes: %v != %v", s.SeqType, s2.SeqType)
+	}
+	return nil
+}
+
+func (s SigChainLocation) Sub1() SigChainLocation {
```

Comment:
Any reason not to just call it "Prev"?

Reply:
I thought that `Prev` might sound like a getter for a field named `Prev` stored in the `SigChainLocation`. There is no such field, I just didn't want confusion about what's in there.